### PR TITLE
Add vue-dnd2

### PR DESCRIPTION
**Pull request by iFgR**
Just add vue-dnd2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -436,6 +436,7 @@
   - [vue-input-tag](https://github.com/matiastucci/vue-input-tag) :bookmark:: Vue.js 2.0 Input Tag Component
   - [vue-cmap](https://github.com/doodlewind/vue-cmap) China map visualizing component for Vue.js 2.0
   - [vue-keyboard](https://github.com/MartyWallace/vue-keyboard) A simple virtual keyboard component for Vue.js 2.0.
+  - [vue-dnd2](https://github.com/iFgR/dnd2) Drag and drop elements sharing information between them. VueJS 2
```
Comment (kazupon) [inline README.md]: I did not find this repo. 404 ...

File: README.md [hunk 2]
```diff
@@ -491,7 +492,8 @@
   - [loopback-vue ★32](https://github.com/qxl1231/generator-loopback-vue) loopback+vue+vue-resource,ionic-app,vue page分页功能,authenticate 权限控制,accesstoken机制,credentials,CI,docker  [qxl1231](https://github.com/qxl1231)
   - [**vue-s3-dropzone**](https://github.com/kfei/vue-s3-dropzone) ![shield](https://img.shields.io/github/stars/kfei/vue-s3-dropzone.svg?style=social&label=Star&maxAge=2592000) A Vue.js drag-and-drop component uploads files to AWS S3 serverlessly
   - [**easy-vue**](https://github.com/TIGERB/easy-vue) ![shield](https://img.shields.io/github/stars/tigerb/easy-vue.svg?style=social&label=Star&maxAge=2592000) a easy example using the vue to implement easy web with vue 2.0, vuex 2.0,
-vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https://github.com/tigerb) 
+
```
Comment (kazupon) [inline README.md]: what mean this new line?"

File: README.md [hunk 3]
```diff
@@ -527,7 +529,9 @@ vue-router 2.0, vue-infinite-scroll 2.0, vue-progressbar 2.0 by [TIGERB](https:/
   - [Vue fullstack](https://github.com/erguotou520/vue-fullstack) - A vue fullstack template to generate a reactivity, realtime, user-friendly backend system. Includes: Vue.js 2, ElementUI, express-jwt, mongodb, etc.
 
   - [Vue SSR Boilerplate](https://github.com/fenivana/vue-ssr-boilerplate) by [@fenivana](https://github.com/fenivana) - Vue.js Server Side Rendering Boilerplate without Polluting Vuex
-  
+
```
Comment (kazupon) [inline README.md]: duplicating ...

**Comment by iFgR** [comment]
Sorry man,
I did some mistakes.
I will create a new PR with right things!

**Pull request closed without merging**